docs/api/driver.md

```diff
@@ -236,6 +236,24 @@ getLogConfig(): LogConfig
 
 Returns the current logging configuration.
 
+### `checkForConfigUpdates`
+
+```ts
+checkForConfigUpdates(): Promise<string | undefined>
```

Comment:
@AlCalzone Forgot to document `silent` var here

Reply:
That's okay. It is only used internally so we don't get duplicate logging output if you first check and then update.